MLP:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: position_controllers/JointPositionController
    joint: front_motor_rev_joint
    pid: {p: 10, i: 0.01, d: 0}

  joint2_position_controller:
    type: position_controllers/JointPositionController
    joint: upper_motor_rev_joint
    pid: {p: 10, i: 0.01, d: 0}

controller_joint_names: ['', 'front_L_motor_rev_joint', 'front_R_motor_joint', 'rear_L_motor_rev_joint', 'rear_R_motor_rev_joint', 'front_motor_rev_joint', 'upper_motor_rev_joint', ]
